autoware.proj.repos

```diff
@@ -47,6 +47,19 @@ repositories:
     type: git
     url: https://github.com/ros2/geometry2.git
     version: foxy
+  vendor/grid_map:
+    type: git
+    url: https://github.com:ANYbotics/grid_map.git
+    version: ros2
+  vendor/filters:
+    type: git
+    url: https://github.com:ros/filters.git
+    version: ros2
+  vendor/orocos_kinematics_dynamics:
+    type: git
+    url: https://github.com:ros2/orocos_kinematics_dynamics.git
+    version: ros2
```

Comment:
These packages are not used by costmap_generator and I don't think we need to include this.
Instead, you have to add perception_pcl which is used by grid_map package https://github.com/ANYbotics/grid_map/blob/ros2/tools/ros2_dependencies.repos
```suggestion
  # dependency for grid_map
  vendor/perception_pcl:
    type: git
    url: https://github.com/ros-perception/perception_pcl.git
    version: foxy-devel
```

Reply:
@mitsudome-r costmap_generator was giving compilation errors related to KDL, I think the KDL was required by PCL.